### PR TITLE
Setup project files

### PR DESCRIPTION
**Pull request by ishimwepati**
Inside this branch, a lot was done, the main features are :

- A HTML file 
- A CSS file
- Configuration of linters
- A professional ReadMe file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,178 @@
...
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Added Linters
+> - Added HTML
+> - Added CSS
```
Comment (usorfaitheloho) [inline README.md:67]: Kindly add features you introduced in this milestone such as ` added toolbar`, `added navigation` etc  so as to give a better explanation to users of the features you introduced in each milestone😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <section class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
+        </section>
```
Comment (usorfaitheloho) [inline index.html]: kindly use a `nav` tag for the outer container of the `hamburger` menu for convention, semantics, and user experience(UX) 😄 

ie 
  >
        <nav class="menu_icon">
                <div></div>
                <div></div>
                <div></div>
        </nav>

File: style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  padding-left: 85%;
+  padding-right: 30%;
+}
+
+.heading_up {
+  color: #172b4d;
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-size: 40px;
+}
```
Comment (usorfaitheloho) [inline style.css]: Kindly use `flex` property to align your header items as it helps with the flexibility of your nav items 😄

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <section class="social_links">
+            <img src="icon/twitter.png" alt=""  class="icon">
+            <img src="icon/Linkedin icon.png" alt=""  class="icon">
+            <img src="icon/medium.png" alt=""  class="icon">
+            <img src="icon/github.png" alt=""  class="icon">
+            <img src="icon/angellist.png" alt=""  class="icon">
+            <br>
+        </section>
+    </nav>
```
Comment (usorfaitheloho) [inline index.html]: To show good semantics and to enable visitors to be able to connect to your social icons, kindly use a `<ul> <li> </li></ul>` tag for your social icon like this :

>

        <ul class="social_links">
           <li>  <a href=""#> </a></li>
          <li>  <a  href="#"> </a></li>
           <li> <a  href="#">    </a></li>
           <li> <a  href="#">  </a></li>
           <li> <a href="#">  </a></li>
            <br>
         </ul>

File: index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    </header>   
+    
+    <nav class="main_section">
+        <section class="home_section">
+            
+            <h1 class="heading_up"><br>Hey there.<br> I’m Ishimwe.</h1>
+            <h1 class="heading_down">I’m a software developer</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+        </section>
```
Comment (usorfaitheloho) [inline index.html]: Please  kindly use a `p` tag for this paragraph to show good semantics and help with Google search optimization 😄

File: index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    
+    <header>
+        <section class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
+        </section>
+    </header>   
+    
+    <nav class="main_section">
```
Comment (usorfaitheloho) [inline index.html]: Please the container of this `headline` section should be a `section` tag and not a `nav` tag as it represents a generic standalone section of your project 😄

File: style.css [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+.social_links {
+  margin-left: 80px;
+  margin-right: 80px;
+  color: #505f79;
+}
+
+.icon {
+  margin: 7px;
+}
```
Comment (usorfaitheloho) [inline style.css:76]: kindly implement the following design changes to enable you have a better UI and good user interaction:

- [x] Your hamburger menu has a bigger width than what we have in the design given, please change it so we have a similar design with yours
- [x] The `font-color` property of your paragraph is different from what we have in the design given, kindly use this         
    ` #172B4D` color for your font in that paragraph

figma design :
image

Your design :
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+            <img src="icon/medium.png" alt=""  class="icon">
+            <img src="icon/github.png" alt=""  class="icon">
+            <img src="icon/angellist.png" alt=""  class="icon">
+            <br>
+        </section>
+    </nav>
+
+    <nav>
+
+    </nav>
```
Comment (usorfaitheloho) [inline index.html]: kindly remove this empty nav tag here so you have a clean code

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @ishimwepati   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ishimwepati) [reply README.md:67]: Ohh thank you

File: index.html (hunk 2 above)
Comment (ishimwepati) [reply index.html]: Good

File: index.html (hunk 4 above)
Comment (ishimwepati) [reply index.html]: Oh Yes, thanks a lot !

File: index.html (hunk 5 above)
Comment (ishimwepati) [reply index.html]: Good

File: index.html (hunk 6 above)
Comment (ishimwepati) [reply index.html]: Great, let me do it!

File: index.html (hunk 8 above)
Comment (ishimwepati) [reply index.html]: Ohh thanks a lot for that guidance.

File: style.css (hunk 7 above)
Comment (ishimwepati) [reply style.css:76]: Wow, thanks for this guidances.

File: style.css (hunk 3 above)
Comment (ishimwepati) [reply style.css]: Thanks a lot

File: index.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <nav class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
```
Comment (jmonto55) [inline index.html]: - [ ] Great work using the `<nav>` bar to wrap your burger menu 💯. However, please kindly ensure to remove all empty `<div>`  tags. 🤓

File: style.css [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+  padding-left: 85%;
+  padding-right: 30%;
+}
+
+.heading_up {
+  color: #172b4d;
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-size: 40px;
+}
```
Comment (jmonto55) [inline style.css]: - [ ] As requested by the previous reviewer, Kindly use flex property to align your header items. ✔️

**Review by jmonto55 — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES 🟡

Hi @ishimwepati 👋🏽, 

Great work on making the changes requested by a previous reviewer 👌
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereGIF


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+                    <a href="#menu" class="box-shadow-menu"></a>
+                </nav>
+        </nav>
+    </header>   
+    
+    <section class="main_section">
+        <section class="home_section">
+            
+            <h1 class="upper_word"><br>Hey there.<br> I’m Ishimwe.</h1>
+            <h1 class="down_word">I’m a software developer</h1>
```
Comment (AbelG101) [inline index.html:24]: - [ ] [OPTIONAL] It is not recommended to use more than one `h1` tag on a landing page. The `h1` tag should be used to indicate the main heading or title of the page and should be unique to that page. Having multiple `h1` tags can confuse search engines and users about the main topic of the page and can negatively impact the page's ranking in search results. 
- [ ] So I'd suggest you place the text "I’m a software developer" in an `h2` tag to follow the proper semantics. 😉

**Review by AbelG101 — APPROVED** [review]
# Hi @ishimwepati 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 11 above)
Comment (ishimwepati) [reply index.html:24]: Thanks

**Pull request merged**